Client/Frontend/Browser/Toolbars/UrlBar/RewardsButton.swift

```diff
@@ -8,31 +8,22 @@ import BraveShared
 
 class RewardsButton: UIButton {
     
-    var isVerified = false {
-        didSet { updateView() }
+    enum IconState {
+        case initial
+        case enabled
+        case disabled
     }
     
-    var notificationCount = 0 {
-        didSet { updateView() }
-    }
-    
-    var forceShowBadge = false {
-        didSet { updateView() }
-    }
-    
-    private let notificationsBadgeView = UIView().then {
-        $0.isUserInteractionEnabled = false
-        $0.backgroundColor = BraveUX.braveOrange
-        $0.frame = CGRect(x: 19, y: 5, width: 12, height: 12)
-        $0.layer.cornerRadius = 6
+    var iconState: IconState = .initial {
+        didSet {
+            updateView()
+        }
     }
     
-    private var checkmarkView = UIImageView().then {
-        let checkmark = #imageLiteral(resourceName: "brave_rewards_verified_badge")
-        $0.image = checkmark
-        $0.frame = CGRect(x: 19, y: 4, width: checkmark.size.width, height: checkmark.size.height)
+    private let lookAtMeBadge = UIImageView(image: #imageLiteral(resourceName: "rewards-look-at-me")).then {
         $0.isHidden = true
         $0.isUserInteractionEnabled = false
+        $0.frame = CGRect(x: 17, y: 7, width: 16, height: 16)
```

Comment:
Can we move them to UX struct like you used in different places?

Reply:
This is only used in a single place so I don't think it needs to live anywhere in particular since its not shared/duplicated